blockboard/src/main/resources/static/js/ajax/postAjax.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @author  Woohyeok Jun <woohyeok.jun@worksmobile.com>
+ * @file    postAjax.js
+ */
+function insertPost(boardID, postTitle, postContent) {
+    $.ajax({
+        type: 'POST',
+        url: "/boards/" + boardID + "/posts/",
+        data: {
+            postTitle: postTitle,
+            postContent: postContent
+        },
+        error: function () {
+            alert("게시글 작성 실패");
+        },
+        success: function () {
+            refreshPostList();
+        }
+    });
+}
+
+function loadPost(boardID, postID) {
+    var post_title = $('#post_title');
+    var editorcontent = $('#editorcontent');
+    var editor = $('#editor');
+    $.ajax({
+        type: 'GET',
+        url: "/boards/" + boardID + "/posts/" + postID + "/editor",
```

Comment:
```suggestion
        url: `/boards/${boardID}/posts/${postID}/editor`,
```
JS에는 이러한 문법도 있어요~
사용 하는 사람 취향인데,, 참고 하면 좋을것 같아요 ㅎㅎ

Reply:
엇...꿀팁 감사합니다!